mapreduce/mapreduce-base/src/main/java/io/tidb/bigdata/mapreduce/tidb/TiDBConfiguration.java

```diff
@@ -46,6 +46,11 @@
   public static final String CLUSTER_TLS_CA = "tidb.cluster-tls-ca";
   public static final String CLUSTER_TLS_KEY = "tidb.cluster-tls-key";
   public static final String CLUSTER_TLS_CERT = "tidb.cluster-tls-cert";
+  public static final String CLUSTER_JKS_ENABLE = "tidb.cluster-jks-enable";
+  public static final String CLUSTER_JKS_KEY_PATH = "tidb.cluster-jks-key-path";
+  public static final String CLUSTER_JKS_KEY_PASSWORD = "tidb.cluster-jks-key-password";
+  public static final String CLUSTER_JKS_TRUST_PATH = "tidb.jks.trust_path";
```

Comment:
All configuration items uses dash however `trust_path` uses underscore instead. We should make it consistent.